frontend/public/components/api-explorer.tsx

```diff
@@ -578,35 +601,37 @@ const APIResourceAccessReview: React.FC<APIResourceTabProps> = ({
           onSelectAll={onSelectAll}
         >
           <CheckBox
-            title="User"
+            title={t('api-explorer~User')}
             active={showUsers}
             number={users.length}
             toggle={toggleShowUsers}
           />
           <CheckBox
-            title="Group"
+            title={t('api-explorer~Group')}
             active={showGroups}
             number={groups.length}
             toggle={toggleShowGroups}
           />
           <CheckBox
-            title="ServiceAccount"
+            title={t('api-explorer~ServiceAccount')}
```

Comment:
It's not clear to me whether we should i18n these. I'm sure Sam will chime in here.

Reply:
@rebeccaalpert I think last i heard these are left as is...